### DISCUSSION TITLE
Software-Blender（动画篇+合成篇） | Zi-Zi's Journey

### DISCUSSION THREAD

**Discussion by giscus** [GZ-Metal-Cell/GZ-Metal-Cell.github.io#349] [Announcements]
# Software-Blender（动画篇+合成篇） | Zi-Zi's Journey

学习自 B 站 UP 主 KurTips。

https://gz-metal-cell.github.io/2023/10/23/Software-Blender%EF%BC%88%E5%8A%A8%E7%94%BB%E7%AF%87+%E5%90%88%E6%88%90%E7%AF%87%EF%BC%89/

**Comment by promefire** [comment]
想问一下关神当年学了多久/(ㄒoㄒ)/~~

**Reply by GZ-Metal-Cell** [reply]
一周左右吧 看这个博客发帖日期也是这样的😵只不过我本科学过3ds max、Pr、spine这些软件可能会快一点
